module/VuFind/src/VuFind/Resolver/Driver/Jop.php

```diff
@@ -50,8 +50,17 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:link_resolver_drivers Wiki
  */
-class Ezb extends AbstractBase
+class Jop extends AbstractBase
 {
+    /**
+     * As the JOP resolver provides also generic labels 'Article', 'Journal'
+     * etc. in element AccessLevel this label can be used as title for
+     * resolver results by setting this variable to 'AccessLevel'
+     *
+     * @var string
+     */
+    protected static $xpathTitleSelector = 'Title';
```

Comment:
Is there a strong reason to make this static, rather than a regular property?

Reply:
In finc we are using a trait and a child class of EZB / JOP with a constructor, in which we do not call the parent constructor. So with a static property or const we would minimize changes.

Reply:
Yes, You were right. I might have thought to cumbersome and can no longer understand my thoughts from back then, so I made xpathTitleSelector a regular property like the others.

Reply:
Great, thanks! I believe this is ready to merge now.